cvat/apps/engine/default_settings.py

```diff
@@ -24,3 +24,8 @@
 """
 Sets the frequency of checking the readiness of the chunk
 """
+
+CVAT_CHUNK_LOCK_ACQUISITION_TIMEOUT = 50
```

Comment:
Could you please clarify the benefit of introducing this constant? It looks like this timeout could be different depending on where the code is executed (but now the same value is used). For instance, in the case of request handling, this value should be less than request timeout (otherwise, we get 504), but in the case of running inside a worker process, I guess this could be more than lock TTL.
So, if we are not planning now to have the different values depending on where a code is running, I guess it's better just to use default values from `get_rq_lock_for_job`.